tests/Tutor.Web.Tests/Integration/TutorTestFactory.cs

```diff
@@ -84,4 +70,18 @@ private static string CreateConnectionStringForTest()
         return
             $"Server={server};Port={port};Database={database};User ID={user};Password={password};Integrated Security={integratedSecurity};Pooling={pooling};";
     }
+
+    private static string CreateConnectionStringForEvents()
```

Comment:
It's really ugly that this method is basically a copy of CreateConnectionStringForTest, but I wasn't sure how to refactor it right now to make it better. 
Also I feel like it would generally be good to separate all environment variables for the event database from environment variables for the regular database, even though most of them would have the same values right now.